src/diff.rs

```diff
@@ -0,0 +1,101 @@
+use crate::undo;
+
+use std::collections::HashMap;
+use std::env;
+
+/// print a diff of the env
+pub fn run(context: bool, color: bool, shadowenv_data: &str) -> i32 {
+    let mut parts = shadowenv_data.splitn(2, ":");
+    let _prev_hash = parts.next();
```

Comment:
I'm sure there's something we could do to the iterator to `drop(1)` but I don't know what it is.